src/components/user-list.js

```diff
@@ -39,6 +35,10 @@ export default class UserList extends React.Component {
         UserStore.removeListener(CHANGE_EVENT, this._onChange);
     }
 
+    handleClickAddUser () {
```

Comment:
(minor)
the expected order of components' s methods is
```js
_getAppState
_onChange
_handleClickAddUser //can be private too
//component live cycle methods
componentDidMount
componentWillUnmount
render //the last one
```
I wrote it only because you replace the `handleClickAddUser` method anyway)